app/models/event_instance.rb

```diff
@@ -20,6 +20,18 @@ class EventInstance < ActiveRecord::Base
 
   validate :dont_update_after_finished, on: :update
 
+  def self.active_hangouts
+    select(&:live?)
+  end
+
+  def self.extract_yt_id(yt_video_url)
```

Comment:
@tansaku Yes, I've added the specs for these. `youtube_rails` looks great and cover more variation of youtube video URLs. Should I use that? One argument against using it is that it will extra overhead.